cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,15 @@
+describe(`Login Test`, () => {
+  it(`should log in with valid credentials`, () => {
+    cy.visit(`https://maglus02.github.io/social-media-client/`);
+    cy.wait(1000);
+    cy.get('.modal-footer button[data-auth="login"]')
+      .should("be.visible")
+      .click();
+    cy.wait(1000);
+    cy.get(`#loginEmail`).type(`santiagotest@noroff.no`);
+    cy.get(`#loginPassword`).type(`Testing123`);
+    cy.get(`button.btn-success`).contains("Login").click();
+    cy.wait(500);
+    cy.get("button.btn-outline-warning").contains("Logout").should("exist");
```

Comment:
It would be more robust to check local storage, as button text/classes/ids may change in future.